### ISSUE TITLE
Plugin for Autokickban when words on a "badword" list are used

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/EV0002#10]
Against Spam attacks.

The bad list should be updatable on the fly (or provide at least a reload feature available via IRC)

**Comment by Ghabry** [comment]
autovoice.rb basicly mitigated it but there is an enhancement suggested here:

https://indieweb.org/IRC#Red_Alert_Mode

1. So like ours but channel mode is +mz
2. EV0001 can still do badword kicks
3. Replay the message when not a badword and +v

Problem: EV0001 logs the spam messages in our log file when we do this :/

**Comment by carstene1ns** [comment]
Yeah, badword filter is just more work, so the autovoice solution was easier for the current attack.
